confgenerator/logging_receivers.go

```diff
@@ -57,6 +57,7 @@ type LoggingReceiverFilesMixin struct {
 	ExcludePaths            []string        `yaml:"exclude_paths,omitempty"`
 	WildcardRefreshInterval *time.Duration  `yaml:"wildcard_refresh_interval,omitempty" validate:"omitempty,min=1s,multipleof_time=1s"`
 	MultilineRules          []MultilineRule `yaml:"-"`
+	BufferInMemory          bool
```

Comment:
Mark it as `yaml:"-"` so it can't be set by user config.

Reply:
Changed in: [f67580f](https://github.com/GoogleCloudPlatform/ops-agent/pull/631/commits/f67580f4f76efa0303dc9986a7867855b006bd27)